extras/local-omi-bt/start.sh

```
@@ -0,0 +1 @@
+uv run --with-requirements requirements.txt python connect-laptop.py
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)